src/Dfc.CourseDirectory.Services/PostCodeSearchCriteria.cs

```diff
@@ -8,38 +8,18 @@ namespace Dfc.CourseDirectory.Services
     public class PostCodeSearchCriteria : ValueObject<PostCodeSearchCriteria>, IPostCodeSearchCriteria
     {
         public string Search { get; }
-        //public int Top { get; }
-        //public int Skip { get; }
-        //public bool Count => true;
-        //public string Filter { get; }
-        //public IEnumerable<LarsSearchFacet> Facets { get; }
 
         public PostCodeSearchCriteria(
             string search)
-            //int top,
-            //int skip,
-            //string filter = null,
-            //IEnumerable<LarsSearchFacet> facets = null)
         {
             Throw.IfNullOrWhiteSpace(search, nameof(search));
-            //Throw.IfLessThan(1, top, nameof(top));
-            //Throw.IfLessThan(0, skip, nameof(skip));
 
             Search = search;
-            //Top = top;
-            //Skip = skip;
-            //Filter = filter;
-            //Facets = facets;
         }
 
         protected override IEnumerable<object> GetEqualityComponents()
         {
-            yield return Search;
-            //yield return Top;
-            //yield return Skip;
-            //yield return Count;
-            //yield return Filter;
-            //yield return Facets;
+            yield return Search;;
```

Comment:
Double semi colon

Reply:
@sbradley090877 I think you might need to pull from master again because Anatoli's PR has just gone in before yours.